website/pages/en/index.js

```diff
@@ -230,29 +203,28 @@ const OpenCollectiveSponsors = ({ language }) => {
             type="opencollective"
             title="Gold Sponsors (Open Collective)"
             tier="gold-sponsors"
-            button={ocButton}
           />
           <SponsorTier
-            type="other"
-            title="Misc Sponsors"
-            tier="other-sponsors"
+            type="opencollective"
+            title="Silver Sponsors (Open Collective)"
```

Comment:
Is this title shown anywhere? If not, we could maybe do something like this: (it's in the "title" of the link, that is shown on hover)

![schermata da 2019-01-08 23-40-56](https://user-images.githubusercontent.com/7000710/50863514-2f8cdf80-139f-11e9-9212-72c705f033a1.jpg)